custom_components/epic_games/__init__.py

```
@@ -17,22 +23,25 @@
-        hass.data[DOMAIN][entry.entry_id] = entry.data
-
-        for platform in PLATFORMS:
-            hass.async_create_task(
-                hass.config_entries.async_forward_entry_setup(entry, platform)
-            )
-    else:
-        raise ConfigEntryAuthFailed("Invalid credentials")
+    )
+
+    if not data:
+        raise ConfigEntryAuthFailed("Invalid credentials or API error")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)